mongodbforms/util.py

```diff
@@ -52,6 +52,10 @@ def load_field_generator():
 def init_document_options(document):
     if not isinstance(document._meta, (DocumentMetaWrapper, LazyDocumentMetaWrapper)):
         document._meta = DocumentMetaWrapper(document)
+    # Workaround for Django 1.7+
+    document._deferred = False
+    # FIXME: Wrong implementation for Relations (https://github.com/django/django/blob/master/django/db/models/base.py#L601)
+    document.serializable_value = lambda self, field_name: self._meta.get_field(field_name)
```

Comment:
I was lazy. This needs to be rewritten as indicated by the FIXME to handle relations properly anyway.